lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +218,23 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        return [ConlluSentence(idx, sentence, []) for idx, sentence in enumerate(split_by_sentence(text))]
 
     def run(self) -> None:
         """
         Performs basic preprocessing and writes processed text to files
         """
+        for key, val in self._corpus.get_articles().items():
+            article = from_raw(val.get_raw_text_path(), val)
```

Comment:
it is already read by corpus manager (!)

Reply:
got it!